ScoutSuite/providers/aws/rules/findings/iam-user-without-mfa.json

```diff
@@ -1,7 +1,15 @@
 {
     "description": "User without MFA",
-    "path": "iam.users.id",
+    "rationale": "<b>Description:</b><br><br>All IAM users should have Multi Factor Authentication (MFA) enabled.",
```

Comment:
Remove `<b>Description:</b><br><br>`

Reply:
This is fixed and will be available in the next commit.